docroot/sites/iisc.uiowa.edu/modules/iisc_core/templates/views-view-unformatted--projects-topic.html.twig

```diff
@@ -0,0 +1,48 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ *
+ * @ingroup themeable
+ */
+#}
+  {% for key,row in rows %}
+    {% set row_classes = [
+      default_row_class ? 'views-row',
+    ] %}
+    {% set media_attributes = media_attributes ?: create_attribute() %}
+    <div{{ row.attributes.addClass(row_classes) }}>
+      {%
+        set classes = [
+        'bg--white',
+        'card--centered',
+        'card--alignment-left',
+        'media--border',
+        'card--stacked ',
+        'card--enclosed',
+        'card__row',
+      ]
+      %}
+
+      {% set projects_card = {
+        'attributes': attributes.addClass(classes),
+        'card_image': view.style_plugin.getField(key, 'field_image'),
+        'card_title': view.style_plugin.getField(key, 'name'),
+        'card_link_url': view.style_plugin.getField(key, 'view_taxonomy_term')|render|striptags|trim,
+        'media_attributes': media_attributes.addClass('card__media--small card__media media--circle'),
```

Comment:
We should move the logic that adds these classes to outside of the for loop since it is not unique per row. This would involve adding the following line after the one above that sets the `media_attributes`:
```twig
{% set media_attributes = media_attributes.addClass('card__media--small card__media media--circle') %}
```

Reply:
I moved these out of the loop, thanks!